guide/daily-spending-wallet/contacts.md

```diff
@@ -38,7 +38,7 @@ imagesSupportedFormats:
       caption: Particular properties of address types can be pointed out right away.
     - file: manual-add-contact-with-on-chain-address
       alt: A contact with an on-chain address assigned
-      caption: A contact with an on-chain address attached. Once a payment is made to the address, it disappears from this list.
+      caption: A contact with an on-chain address attached. Once a payment is made to the address, it disappears from this list as it should only be used once for best privacy.
```

Comment:
For the images, are we going to link each frame directly to the corresponding frame in the UI kit as Milli suggested?

Reply:
The UI kit is not set up for this. We'll have to deal with this as its own project at some point. Without thinking this through and having a proper process, we'll end up with tons of broken links quickly.

Reply:
Alright we'll tackle that later then